src/app/complex/complex-results/complex-navigator/table-structure/table-header/table-header.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="Complex-navigator">
+  <table class="table CN-table">
+    <!-- When table is not overflowing -->
+    <thead class="tableHead" *ngIf="complexes.length<=6">
+    <tr>
+      <th class="interactorsHeader horizontal"
+          [style]="isInteractorSortingSet() ? 'min-width: 135px; max-width: 135px;   border-bottom: 3px solid #007c82;':''">
+        Interactors
+      </th>
+      <th class="horizontal" *ngFor="let complex of complexes">
+        <a [routerLink]="['/complex', complex.complexAC]"
+           title="{{complex.complexAC}}"
+           target="_blank">
+          <div>
+            <span title="{{ complex.complexName }} - {{ complex.complexAC}}">
+              <div class="line"></div>
+                <div class="horizontal-label">
+                  <i class="{{iconOrganism(complex.organismName)}}"></i>
+                  <i class="icon icon-generic small" data-icon="x"></i>
+                  {{ complex.complexName }}
+                </div>
+            </span>
+          </div>
+        </a>
+      </th>
+    </tr>
+    </thead>
+    <!-- When table is overflowing -->
+    <thead class="tableHeadOverflow" *ngIf="complexes.length>6">
+    <tr [style.--cols]="complexes.length"
+        [style.--intHeader]="isInteractorSortingSet() ? '156px':''">
+      <th class="interactorsHeader"
+          [style]="isInteractorSortingSet() ? 'min-width: 156px; max-width: 156px;':''">
+      </th>
+      <th class="rotate" *ngFor="let complex of complexes"
+          title="{{ complex.complexName }} - {{ complex.complexAC}}">
+        <a [routerLink]="['/complex', complex.complexAC]"
+           title="{{ complex.complexName }} - {{ complex.complexAC}}"
+           target="_blank">
+          <div>
+            <span>
+              <div class="tilted-label" title="{{ complex.complexName }} - {{ complex.complexAC}}">
+                <i class="icon icon-generic small" data-icon="x"
+                   title="{{ complex.complexName }} - {{ complex.complexAC}}"></i>
+                {{ complex.complexName }}
+              </div>
+            </span>
+          </div>
+        </a>
+      </th>
+      <div class="spaceHolder"></div>
+    </tr>
+    <tr [style.--cols]="complexes.length" [style.--intHeader]="isInteractorSortingSet() ? '156px':''">
```

Comment:
Should we have `*ngIf="organismIconDisplay"` here? Not sure about it, but if we add it, we then also need to add it in line 18.